### PR TITLE
Fix a few bugs with some stripes pipe handlers

### PR DESCRIPTION
**Pull request by iTitus**

### CODE REVIEW

File: common/buildcraft/transport/BCTransportRegistries.java [hunk 1]
```diff
@@ -67,12 +67,13 @@ public static void init() {
...
-        StripesHandlerDispenser.ITEM_CLASSES.add(ItemBucket.class);
-        StripesHandlerDispenser.ITEM_CLASSES.add(ItemMinecart.class);
+        // For testing
+        //StripesHandlerDispenser.ITEM_CLASSES.add(ItemBucket.class);
+        //StripesHandlerDispenser.ITEM_CLASSES.add(ItemMinecart.class);
+
         // StripesHandlerRightClick.items.add(Items.EGG);
         // StripesHandlerRightClick.items.add(Items.SNOWBALL);
         // StripesHandlerRightClick.items.add(Items.EXPERIENCE_BOTTLE);
-        StripesHandlerUse.ITEMS.add(Items.FIREWORKS);
```
Comment (iTitus) [inline common/buildcraft/transport/BCTransportRegistries.java]: This case is fulfilled by the dispenser handler

File: common/buildcraft/transport/BCTransportRegistries.java [hunk 2]
```diff
@@ -67,12 +67,13 @@ public static void init() {
         PipeApi.stripeRegistry.addHandler(StripesHandlerPlaceBlock.INSTANCE, EnumHandlerPriority.LOW);
         PipeApi.stripeRegistry.addHandler(StripesHandlerUse.INSTANCE, EnumHandlerPriority.LOW);
 
-        StripesHandlerDispenser.ITEM_CLASSES.add(ItemBucket.class);
-        StripesHandlerDispenser.ITEM_CLASSES.add(ItemMinecart.class);
+        // For testing
```
Comment (iTitus) [inline common/buildcraft/transport/BCTransportRegistries.java]: The ITEM_CLASSES list is only used in a commented out section which has a for testing remark

File: common/buildcraft/transport/stripes/StripesHandlerHoe.java [hunk 3]
```diff
@@ -28,29 +28,39 @@ public boolean handle(World world,
...
+                EnumHand.MAIN_HAND,
+                EnumFacing.UP,
+                0.0f,
+                0.0f,
+                0.0f
+        ) != EnumActionResult.PASS) {
+            return true;
+        }
+
+        if (direction != EnumFacing.UP && stack.onItemUse(
```
Comment (iTitus) [inline common/buildcraft/transport/stripes/StripesHandlerHoe.java]: The hoe shouldn't try to till the pipe itself

File: common/buildcraft/transport/stripes/StripesHandlerShears.java [hunk 4]
```diff
@@ -46,8 +48,9 @@ public boolean handle(World world,
                 if (stack.attemptDamageItem(1, player.getRNG())) {
                     stack.shrink(1);
                 }
+                world.setBlockState(pos, Blocks.AIR.getDefaultState(), 11); // Might become obsolete in 1.12+
```
Comment (iTitus) [inline common/buildcraft/transport/stripes/StripesHandlerShears.java]: See the ItemShears implementation

File: common/buildcraft/transport/stripes/StripesHandlerShears.java [hunk 5]
```diff
@@ -46,8 +48,9 @@ public boolean handle(World world,
...
                 for (ItemStack dropStack : drops) {
-                    activator.sendItem(dropStack, direction.getOpposite());
```
Comment (iTitus) [inline common/buildcraft/transport/stripes/StripesHandlerShears.java]: This caused the web to get placed instantly again

File: common/buildcraft/transport/stripes/StripesHandlerMinecartDestroy.java [hunk 6]
```diff
@@ -43,13 +43,13 @@ public boolean handle(World world, BlockPos pos, EnumFacing direction, EntityPla
...
-                            activator.sendItem(s, direction.getOpposite());
+                            activator.sendItem(s, direction);
                         }
                     }
                 }
             }
             cart.setDead();
-            activator.sendItem(StackUtil.asNonNull(cart.getCartItem()), direction.getOpposite());
+            activator.sendItem(StackUtil.asNonNull(cart.getCartItem()), direction);
             return true;
```
Comment (AlexIIL) [inline common/buildcraft/transport/stripes/StripesHandlerMinecartDestroy.java]: Unfortunately I can't test this as block breaking is broken... oh well. I'll have to look at that later I suppose.

**Comment by AlexIIL** [comment]
Thanks!

**Pull request merged**